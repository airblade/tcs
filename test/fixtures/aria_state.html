<div aria-checked="true" class="aria-checked:bg-sky-700 bg-gray-600">
</div>
