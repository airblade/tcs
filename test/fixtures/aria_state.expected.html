<div aria-checked="true" class="bg-gray-600 aria-checked:bg-sky-700">
</div>
